tools/generation/lib/template.py

```diff
@@ -161,8 +161,9 @@ def _frontmatter(self, case_filename, case_values):
         features += case_values['meta'].get('features', [])
         features += self.attribs['meta'].get('features', [])
         features = list(OrderedDict.fromkeys(features))
+        initial_space_pattern = re.compile(r'\n\s*')
         if len(features):
-            lines += ['features: ' + re.sub('\n\s*', ' ', yaml.dump(features, default_flow_style=True).strip())]
+            lines += ['features: ' + re.sub(initial_space_pattern, ' ', yaml.dump(features, default_flow_style=True).strip())]
```

Comment:
Good point!

The proof is in the pudding, so I deleted all the generated files and regenerated them with this modification; turns out `yaml.dump` wraps long lines at 80 by default, so that's what this regexp was undoing. I added `width=float('inf')` and that seems sufficient to reproduce the original generated files.